src/sheerwater_benchmarking/data.py

```diff
@@ -1,26 +1,17 @@
-
 import dask
 import cdsapi
-import argparse
-import json
-import pathlib
-import sys
-import time
 import os
-import subprocess
 import xarray as xr
 import dateparser
-from datetime import datetime
-from sheerwater_benchmarking.utils.secrets import cdsapi_secret
-from sheerwater_benchmarking.utils.remote import dask_remote
-from sheerwater_benchmarking.utils.caching import cacheable
+from sheerwater_benchmarking.utils import cdsapi_secret, dask_remote, cacheable, get_grid
 
-@cacheable(data_type='array', immutable_args=['grid'])
-def land_sea_mask(grid=1.5):
 
+@cacheable(data_type='array', cache_args=['grid'])
+def land_sea_mask(grid="global1_5"):
```

Comment:
Changed to a named grid

Reply:
I'm generally fine with this, but realistically I feel like we can't make changes like this without some cache management tool because it invalidates all of our caches which took a long time to make.